testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -298,6 +309,104 @@ impl TxEmitter {
         })
     }
 
+    pub async fn load_dd_account(&self, instance: &Instance) -> Result<AccountData> {
+        let client = instance.json_rpc_client();
+        let mint_key: Ed25519PrivateKey = generate_key::load_key(DD_KEY);
+        let mint_key_pair: KeyPair<Ed25519PrivateKey, Ed25519PublicKey> = KeyPair::from(mint_key);
+        let address = libra_types::account_address::from_public_key(&mint_key_pair.public_key);
+        let sequence_number = query_sequence_numbers(&client, &[address])
+            .await
+            .map_err(|e| {
+                format_err!(
+                    "query_sequence_numbers on {:?} for dd account failed: {}",
+                    client,
+                    e
+                )
+            })?[0];
+        Ok(AccountData {
+            address,
+            key_pair: mint_key_pair.clone(),
+            sequence_number,
+        })
+    }
+
+    pub async fn load_vasp_account(&self, instance: &Instance) -> Result<AccountData> {
+        let client = instance.json_rpc_client();
+        let mint_key: Ed25519PrivateKey = generate_key::load_key(VASP_KEY);
+        let mint_key_pair: KeyPair<Ed25519PrivateKey, Ed25519PublicKey> = KeyPair::from(mint_key);
+        let address = libra_types::account_address::from_public_key(&mint_key_pair.public_key);
+        let sequence_number = query_sequence_numbers(&client, &[address])
+            .await
+            .map_err(|e| {
+                format_err!(
+                    "query_sequence_numbers on {:?} for dd account failed: {}",
+                    client,
+                    e
+                )
+            })?[0];
+        Ok(AccountData {
+            address,
+            key_pair: mint_key_pair.clone(),
+            sequence_number,
+        })
+    }
+
+    pub async fn mint_faucet_account(
+        &self,
+        instances: &Vec<Instance>,
+        coins_total: u64,
+    ) -> Result<AccountData> {
+        let faucet_account = if !self.premainnet {
+            info!("Creating and minting faucet account");
+            let mut account = self
+                .load_faucet_account(self.pick_mint_instance(instances))
+                .await?;
+            let mint_txn = gen_mint_request(&mut account, coins_total, self.chain_id);
+            execute_and_wait_transactions(
+                &mut self.pick_mint_client(instances),
+                &mut account,
+                vec![mint_txn],
+            )
+            .await
+            .map_err(|e| format_err!("Failed to mint into faucet account: {}", e))?;
+            account
+        } else {
+            info!("Loading faucet account from DD account");
+            self.load_dd_account(self.pick_mint_instance(&instances))
```

Comment:
Can we query balance for dd account and print something like
```
DD account has {} coins, requested {coins_total} coins
```
Just so we can triage if there is some issue and DD account runs low balance?